packages/BarChart/storiesConstants.js

```diff
@@ -36,3 +36,194 @@ for (var i = 3; i <= 12; i++) {
     data: getRandomArray(6, 0, 50)
   })
 }
+
+let annotationChartLabels = []
+for (i = 5; i <= 23; i++) {
+  const leadingZero = (number) => number < 10 ? '0' : ''
+  annotationChartLabels.push(`20${leadingZero(i)}${i}-${leadingZero(i + 1)}${i + 1}`)
+}
+export const annotationChartData = {
+  labels: annotationChartLabels,
+  datasets: [
+    {
+      label: 'Dataset 1',
+      data: [8, 9, 9, 47, 49, 55, 60, 7, 8, 9, 45, 55, 60, 52, 58, 69, 63, 60, 65]
+    }
+  ]
+}
+
+let annotationChartColors = []
+let annotationChartHoverColors = []
+for (i = 5; i <= 23; i++) {
+  annotationChartColors.push(i < 20 ? '#f5333f' : '#737679')
+  annotationChartHoverColors.push(i < 20 ? '#f7666f' : '#abacae')
+}
+export const annotationChartCustomStyles = [
+  {
+    backgroundColor: annotationChartColors,
+    hoverBackgroundColor: annotationChartHoverColors
+  }
+]
+
+export const annotation = {
+  annotations: [
```

Comment:
@hgaitanis if you can add some docs for how to add annotations, that will be good. Otherwise others may have no clue about it. I guess it can be just a link to https://www.chartjs.org/chartjs-plugin-annotation/ docs somewhere.

Reply:
Yes, you are right! I have already added that link in the prop description